### DISCUSSION TITLE
Sugestion - Create Subpages / Add Internal Page Support and Routing

### DISCUSSION THREAD

**Discussion by gustavocs97** [JulianPrieber/littlelink-custom#814] [Feature requests]
### Description

The Linkstack project currently allows users to create a list of links displayed on a profile page (e.g., [https://links.linkstack.com/@brand](https://links.linkstack.com/@brand)). I propose adding a new feature to support internal pages with structured URLs, such as `https://links.linkstack.com/@brand/p/test1`.

#### Proposed Changes

1. **Routing System**:
   - Implement a routing system to handle internal URLs, enabling pages like `https://links.linkstack.com/@brand/p/test1`.

2. **User Interface**:
   - Add an option in the user dashboard to create and manage internal pages.
   - Allow users to customize the content and layout of these pages using HTML, CSS, and JS.

3. **Admin Control**:
   - Introduce an admin feature to enable or disable this functionality for individual users. This allows selective access to internal pages.

#### Why?

Supporting internal pages offers a variety of new possibilities. Users can leverage this feature for:
- Subpages for booking or payment systems
- Embedded forms and additional content
- Enhanced profile customization

This enhancement aims to provide greater flexibility and functionality for users, expanding the use cases for Linkstack.

**Comment by chris2deuce** [comment]
This request sounds really interesting.
